src/Core/Mesh/TopologicalTriMesh/TopologicalMesh.cpp

```diff
@@ -208,10 +206,6 @@ TriangleMesh TopologicalMesh::toTriangleMesh() {
     for ( auto oh : m_vec4Pph )
         addAttribPairToCore( out, this, oh, vprop_vec4 );
 
-    request_face_normals();
-    request_vertex_normals();
-    update_vertex_normals();
-
     // iterator over all faces
```

Comment:
I would suggest to also remove these from the Subdividers.
However, there is some issue with the normals there.